src/NetworkFailureToast.tsx

```diff
@@ -85,9 +87,30 @@ export default class NetworkFailureToast extends React.PureComponent<
   }
 
   handleConnectivityChange = (isConnected: boolean) => {
-    this.setState({
-      isNetworkConnected: isConnected,
-    });
+    if (!isConnected) {
+      this.setState(
+        {
+          isHidden: false,
+        },
+        () => {
+          this.setState({
+            isNetworkConnected: isConnected,
+          });
+        }
+      );
```

Comment:
Please add comment to document when separate setState instead of setting all state at one time